src/components/CallToAction.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Link } from "gatsby";
+
+const CallToAction = () => (
+  <section>
+    <h1>Have you completed the pre-onboarding checklist?</h1>
+    <Link to="">Start onboarding today!</Link>
```

Comment:
Same as above, let's make both the link and button text a prop so this component is reusable.